src/clj/commiteth/routes/services.clj

```diff
@@ -234,6 +235,14 @@
                               (if (= 1 result)
                                 (ok)
                                 (internal-server-error)))))
+
+                    (POST "/hidden" []
```

Comment:
How about using just one endpoint to update data for the user? Before we just had the address, but now this too, and in the future probably more things.

Reply:
Agree, will merge those two handlers.

Reply:
Talk to @tpatja first, since he said [here](https://github.com/status-im/open-bounty/pull/199#issuecomment-354154249) that he was already implementing this feature.

Reply:
sure, let me ask here
@tpatja where did you update that handler? In you local branch?

Reply:
Like I said in my previous comment, I implemented this whole feature (backend and frontend changes) already last week. My changes are still on my local repo, will clean up and push once I return to work. Will be on holiday until Jan 8th.

Reply:
I see, that sounds great. But still, I'll refactor that PR to make it satisfy your and @pablanopete suggestions.